e2e/js/js-cdn/src/app/app.element.ts

```diff
@@ -0,0 +1,87 @@
+const SDK_KEY =
+    process.env.DEVCYCLE_CLIENT_SDK_KEY || '<DEVCYCLE_CLIENT_SDK_KEY>'
```

Comment:
does this actually pick up the env var? Nx seems to do dumb things to env vars that prevent them from making it in a lot of the time

Reply:
how did you get it to work for next?

Reply:
I'm trying to copy that implementation but the test is timing out, and I assume this is the reason

Reply:
it lets "NEXT_PUBLIC_" variables go through

Reply:
based on the error Im seeing in my playwright log it seems like not even NEXT_PUBLIC_ is working ```{"message":"Only 'client', 'dvc_client' keys are supported by this API. Invalid key: <DEVCYCLE_CLIENT_SDK_KEY>","statusCode":401}```